skymp5-server/cpp/server_guest_lib/PapyrusGame.cpp

```diff
@@ -110,3 +110,56 @@ void PapyrusGame::RaceMenuHelper(VarValue& self, const char* funcName,
     SpSnippet(GetName(), funcName, serializedArgs.data()).Execute(actor);
   }
 }
+
+VarValue PapyrusGame::GetForm(
```

Comment:
Original function was able to return both espm and form GameObjects. please port this functionality as it was.

also do not copy-paste code here please

if you want to keep maxId check, please write meaningful logs.